guides/common/modules/proc_configuring-tracer-on-a-host.adoc

```diff
@@ -1,18 +1,31 @@
-[id="enabling-tracer-on-a-host_{context}"]
-= Enabling Tracer on a host
+[id="Configuring_Tracer_on_a_host_{context}"]
+= Configuring Tracer on a host
 
-Use this procedure to enable Tracer on {Project} and access Traces.
-Tracer displays a list of services and applications that need to be restarted.
+You can install Tracer on {ProjectName} and access Traces.
+Tracer displays a list of services and applications that are outdated and need to be restarted.
 Traces is the output generated by Tracer in the {ProjectWebUI}.
 
 .Prerequisites
+* The host is registered to {ProjectName}.
 include::snip_prerequisite-project-client-repository-enabled.adoc[]
 * Remote execution is enabled.
 
 .Procedure
+
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
 . Click the name of the host you want to modify.
 . On the *Traces* tab, click *Enable Traces*.
 . Select the provider to install `katello-host-tools-tracer` from the list.
++
+Alternatively, install Tracer using the CLI:
++
+----
+# yum install katello-host-tools-tracer
+----
 . Click *Enable Tracer*.
 You get a REX job notification after the remote execution job is complete.
```

Comment:
This step looks like it doesn't belong to the CLI procedure. It should probably go above, together with the web UI procedure.

Reply:
Deleted the CLI alternative altogether.